packages/launchpad/src/setup-actions.ts

```diff
@@ -113,11 +112,6 @@ export const setUpActionsForTasks = ( {
 				case 'site_launched':
 				case 'blog_launched':
 					action = async () => {
-						await wpcomRequest( {
-							path: `/sites/${ siteSlug }/launch`,
-							apiVersion: '1.1',
-							method: 'post',
-						} );
```

Comment:
Let's reduce the launchpad actions logic. It should only send `onSiteLaunched` events. We moved this logic to the action `launchSiteOrRedirectToLaunchSignupFlow` [in this call](https://github.com/Automattic/wp-calypso/pull/104868/files#diff-44ba101433306b81ab621282570e93ccf7232453f3cdd3cd1b5bfb04ad3d6a5cR15).